packages/cli/src/generators/app.ts

```diff
@@ -64,7 +68,7 @@ class AppGenerator extends Generator<AppGeneratorOptions> {
 
     const result = spawn.sync(this.options.yarn ? 'yarn' : 'npm', ['install'], {stdio: 'inherit'})
     if (result.status !== 0) {
-      throw new Error()
+      throw new Error(`Failed to ${this.options.yarn ? 'yarn' : 'npm'} install`)
```

Comment:
Since we now use the yarn/npm split in a few places, would it be possible to pull that logic out into a constant? It'll prevent any chance of us (most likely me 😬) messing it up in the future

Reply:
Done 77d4a5c